python_src/src/lamp_py/performance_manager/gtfs_utils.py

```diff
@@ -31,8 +31,14 @@ def start_time_to_seconds(
     """
     if time is None:
         return time
-    (hour, minute, second) = time.split(":")
-    return int(hour) * 3600 + int(minute) * 60 + int(second)
+
+    try:
+        (hour, minute, second) = time.split(":")
+        return int(hour) * 3600 + int(minute) * 60 + int(second)
+    except ValueError:
+        # some older files have the start time already formatted as seconds
+        # after midnight. in those cases, convert to an int and pass through.
+        return int(time)
```

Comment:
just convert if we can't split the time. this should still surface other errors.